worlds/witness/options.py

```diff
@@ -151,50 +178,61 @@ class VictoryCondition(Choice):
 
 
 class PuzzleRandomization(Choice):
-    """Puzzles in this randomizer are randomly generated. This setting changes the difficulty/types of puzzles."""
+    """
+    Puzzles in this randomizer are randomly generated. This setting changes the difficulty/types of puzzles.
+    """
     display_name = "Puzzle Randomization"
     option_sigma_normal = 0
     option_sigma_expert = 1
     option_none = 2
 
 
 class MountainLasers(Range):
-    """Sets the amount of lasers required to enter the Mountain.
-    If set to a higher amount than 7, the mountaintop box will be slightly rotated to make it possible to solve without
-    the hatch being opened.
-    This change will also be applied logically to the long solution ("Challenge Lasers" setting)."""
+    """
+    Sets the amount of lasers required to enter the Mountain.
+    If set to a higher amount than 7, the mountaintop box will be slightly rotated to make it possible to solve without the hatch being opened.
+    This change will also be applied logically to the long solution ("Challenge Lasers" setting).
+    """
     display_name = "Required Lasers for Mountain Entry"
     range_start = 1
     range_end = 11
     default = 7
 
 
 class ChallengeLasers(Range):
-    """Sets the amount of beams required to enter the Caves through the Mountain Bottom Floor Discard."""
+    """
+    Sets the amount of beams required to enter the Caves through the Mountain Bottom Floor Discard.

```

Comment:
It may be good to explicitly state that this also impacts the Challenge Timer Panel
```suggestion
    Sets the number of lasers required to enter the Caves through the Mountain Bottom Floor Discard.
```

Reply:
How would you word that?

Reply:
Probably just add "and to start The Challenge" at the end?